server/api/paths.js

```diff
@@ -82,7 +82,7 @@ router.get('/:pathUid', async (req, res, next) => {
     const param = req.params.pathUid
 
     const query = `
-    MATCH (p:Path) WHERE p.uid = {uid}
+    WHERE c.name = {category} AND p.status = "public"
```

Comment:
This route is `api/paths/:uid` and it doesn't seem like it should be modified with `status: 'public'`.  It should just return a single path by `uid`.